configuration.py

```diff
@@ -32,42 +32,25 @@
 class Configuration:
     """Class that stores all parameters used by ABCD."""
 
-    def __init__(self, 
-        project_id,
-        project_zone:str,
-        bucket_name,
-        knowledge_graph_api_key:str,
-        use_annotations:bool,
-        use_llms:bool,
-        assessment_file:str,
-        verbose:bool,
-    ):
+    def __init__(self):
         """Initialize with only the required parameters.
 
           We set all optional parameter defaults in this class because
           we do not want anyone importing the global constants. Hence
           no global variables for hard coded values by design.
-
-        Args:
-          project_id: Google Cloud Project ID
-          project_zone: Google Cloud Project zone (us-central1 if None)
-          bucket_name: Google Cloud Storage Bucket name (not uri) 
-          knowledge_graph_api_key: Google Cloud API Key (limit this)
-          use_annotations: Use video annotation AI.
-          use_llms: Use LLM AI.
-          assessment_file: If present, results will be written to the file path.
-          verbose: Turn on extra debug and execution prints.
         """
-        self.project_id = project_id 
-        self.project_zone = project_zone or "us-central1"
-        self.bucket_name = bucket_name
-        self.knowledge_graph_api_key = knowledge_graph_api_key
-        self.use_annotations = use_annotations
-        self.use_llms = use_llms
-        self.assessment_file = assessment_file
-        self.verbose = verbose
-
-        self.annotation_path = f"gs://{bucket_name}/ABCD/"
+        # set parameters
+        self.project_id = "" 
```

Comment:
You should try to differentiate public v. private attributes w/ underscore (`_`) as necessary.

Reply:
Yes, in this case they are all public.